src/components/FormButton.vue

```diff
@@ -0,0 +1,27 @@
+<template>
+  <button
+    class="m-1 w-1/2 h-12 bg-purple-500 hover:bg-purple-700 text-white font-bold py-2 px-4 rounded"
+  >{{ message }}</button>
+</template>
+
+<script>
+export default {
+  name: "FormButton",
+  props: ["message", "color"],
```

Comment:
I see I added a color prop here, any way we can make the prop actually adjust the color in a Tailwind-ish style ? That way we can throw this into a component library.

Actually on second thought, not worth the effort. But you should have an idea about how you would do this so you're prepared in the future.

Reply:
@jseanpatel Forget it.